src/components/Site/clientSideRedirections.js

```diff
@@ -0,0 +1,13 @@
+// RedirectWebpackPlugin won't work for URI fragments
+// hence we redirect them with client side script
+export default function (location) {
```

Comment:
I'm adding a function for future client slide redirections. Otherwise we might bloat `Site.jsx`.

Reply:
Awesome, Thanks!